libraries/ts-command-line/src/providers/CommandLineAction.ts

```diff
@@ -87,6 +88,21 @@ export abstract class CommandLineAction extends CommandLineParameterProvider {
       description: this.documentation
     });
 
+    // Monkey-patch the error handling for the action parser
+    this._argumentParser.exit = (status: number, message: string) => {
```

Comment:
The throw gets handled gracefully by the CommandLineParser. This was resolving an issue where some situations would cause argparse to `process.exit(...)` after printing to the console, which is what we try to avoid with our original handling of the exit call, so I'm just extending that behavior to the action argument parsers.